iree/tools/iree-benchmark-module-main.cc

```diff
@@ -165,9 +105,111 @@ void RegisterModuleBenchmarks(
       // we can make this setting configurable with a custom command line flag.
       ->Unit(benchmark::kMillisecond);
 }
+
+Status GetModuleContentsFromFlags(std::string& module_data) {
+  auto module_file = absl::GetFlag(FLAGS_module_file);
+  IREE_ASSIGN_OR_RETURN(module_data, file_io::GetFileContents(module_file));
+  return iree::OkStatus();
+}
+
+Status PrepareIREEVMResources(iree_vm_instance_t** instance,
+                              iree_hal_device_t** device,
+                              iree_vm_module_t** hal_module,
+                              iree_vm_context_t** context,
+                              iree_vm_module_t** input_module,
```

Comment:
not needed as part of this PR, but with the vm::ref utility this can all be made safer
example: 
```
        vm::ref<iree_vm_list_t> inputs;
        IREE_CHECK_OK(iree_vm_list_create(/*element_type=*/nullptr, 6,
                                          iree_allocator_system(), &inputs));
```

(basically, vm::ref is kind of like a shared_ptr, and can be passed to functions like create() directly)

Reply:
Use a class to handle resources, so we don't need to pass these.